File: 2026-08/src/day_28/prms_120888/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

string solution(string my_string) {
    string s;
    set<char> seen;

    for (char c : my_string) {
        if (seen.contains(c)) continue;
        s += c;
        seen.insert(c);
    }

    return s;
}

File: 2026-08/src/day_28/prms_120888/Solution.java
package day_28.prms_120888;

import java.util.*;

class Solution {
    public String solution(String my_string) {
        StringBuilder sb = new StringBuilder();
        Set<Character> seen = new HashSet<>();

        for (char c : my_string.toCharArray()) {
            if (seen.add(c)) sb.append(c);
        }

        return sb.toString();
    }
}
